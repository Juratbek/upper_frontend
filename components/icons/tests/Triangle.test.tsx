import { render, screen } from '@testing-library/react';

import { TriangleIcon } from '../Triangle';

describe('Triangle icon', () => {
  it('renders the icon', () => {
    render(<TriangleIcon />);
    const icon = screen.getByRole('icon');
    expect(icon).toMatchSnapshot();
  });
});
