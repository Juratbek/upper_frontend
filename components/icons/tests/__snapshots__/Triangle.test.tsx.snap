// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Triangle icon > renders the icon 1`] = `
<svg
  fill="none"
  role="icon"
  viewBox="0 0 13 16"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M12 6.26795C13.3333 7.03775 13.3333 8.96225 12 9.73205L3 14.9282C1.66667 15.698 0 14.7358 0 13.1962L0 2.80385C0 1.26425 1.66667 0.301996 3 1.0718L12 6.26795Z"
    fill="#D9D9D9"
  />
</svg>
`;
